Home:
  x: 0.00
  y: 0.00

Table1:
  x: -2.00
  y: 0.00

Table2:
  x: 0.00
  y: 2.00

Table3:
  x: 2.00
  y: 0.00

Table4:
  x: 0.00
  y: -2.00
